buildpacks/nodejs-function-invoker/src/function.rs

```diff
@@ -34,6 +34,30 @@ where
         .and_then(|path| path.exists().then_some(path).ok_or(MainError::MissingFile))
 }
 
+pub fn get_declared_runtime_package<P>(
```

Comment:
I know I've seen package declarations that look like this:`"@hk/sf-fx-runtime": "npm:@heroku/sf-fx-runtime-nodejs@0.12.0". It doesn't look like this code would support that. This is obviously unlikely, but would that be supportable by using something like `npm ls` instead of parsing the package.json ourselves?

Reply:
This isn't a blocking comment, btw. This seems very unlikely and a pretty weird use case.

Reply:
@joshwlewis should we support this edge case?  the first time i'd ever seen that style in the wild is with the `@hk/sf-fx-runtime` declaration which makes me think it's fairly uncommon.

but, if you do think we should support this, the value contains the actual package name so we could do some deconstructing of that value instead of using `npm ls`.

Reply:
No, I don't really think it's something to support. I was just thinking `npm ls` might be more authoritative in some less common scenarios, since it'd be leveraging npm's package.json, package-lock.json, npm-shrinkwrap.json parsing.